cmp/form/Label.js

```diff
@@ -5,21 +5,25 @@
  * Copyright © 2018 Extremely Heavy Industries Inc.
  */
 
+import {PropTypes as PT} from 'prop-types';
 import {hoistComponent, elemFactory} from 'hoist/core';
 import {div} from 'hoist/layout';
 
 import {HoistField} from './HoistField';
+
 /**
  * A simple label for a form.
- *
- * @prop children, text to display
- * @prop style
- * @prop className
- * @prop width, width of field, in pixels
  */
 @hoistComponent()
 export class Label extends HoistField {
 
+    static propTypes = {
+        children: PT.node,
+        style: PT.object,
+        className: PT.string,
+        width: PT.number
```

Comment:
Same question re. "standard" props that apply to almost all components, unless there is something special about the way they are processed in a given comp that merits a comment. 

I am not positive I have the right mental model about some of these things so happy to discuss. Also not sure how this all interacts with the "delegateProps" concept, which I just discussed with Lee earlier today (and still need to review).